### PR TITLE
update docs for post sprint 44 release

### PR DESCRIPTION
**Pull request by RichardEustace**

### CODE REVIEW

File: docs/api-documentation/V1.19/index.md [hunk 1]
```diff
@@ -0,0 +1,3480 @@
+---
+layout: default
+title: API specification V1.18
+---
+# API specification
+{: .govuk-heading-xl}
+
+Version 1.18
```
Comment (AgnesAnn) [inline docs/api-documentation/V1.19/index.md]: Version 1.19

File: docs/api-documentation/V1.19/index.md [hunk 2]
```diff
@@ -0,0 +1,3480 @@
+---
+layout: default
+title: API specification V1.18
```
Comment (AgnesAnn) [inline docs/api-documentation/V1.19/index.md]: V1.19

File: docs/api-documentation/index.html [hunk 3]
```diff
@@ -64,8 +64,8 @@ <h1 class="govuk-heading-m">Releases</h1>
 
 <ul class="govuk-list govuk-list--bullet">
   <li>
-    <a class="govuk-link" href="{{ site.baseurl }}/api-documentation/V1.18/">
-      Version 1.18 - Due for release to SANDBOX 19/03/2020
```
Comment (matthewdoreilly) [inline docs/api-documentation/index.html]: Just curious, will V1.18 be the Sprint 45.1 release? And then there'll be a V1.19 for the Sprint 45.2 release?

Comment (RichardEustace) [reply docs/api-documentation/index.html]: 18 is just done (44), 19 will be 45.1 and 20 will be 45.2

Comment (matthewdoreilly) [reply docs/api-documentation/index.html]: Ah sorry, was a typo in my last message, meant V1.19 -> Sprint 45.1 and V.20 -> Sprint 45.2. Cheers for confirming 🙂

**Pull request merged**